model/Entity/TaskLogEntity.php

```diff
@@ -55,43 +56,49 @@ class TaskLogEntity implements JsonSerializable
 
     /** @var  DateTime */
     private $updatedAt;
+    /**
+     * @var null
+     */
+    private $category;
 
     /**
      * TaskLogEntity constructor.
      *
      * @param string                   $id
      * @param string                   $taskName
+     * @param TaskLogCategorizedStatus $status
      * @param array                    $parameters
      * @param string                   $label
-     * @param TaskLogCategorizedStatus $status
      * @param string                   $owner
-     * @param Report                   $report
-     * @param DateTime        $createdAt
-     * @param DateTime        $updatedAt
+     * @param DateTime|null            $createdAt
```

Comment:
In which case the createdAt it's null, this is not a mandatory value? because you need to know the creation timestamp right? The frontend needs this all the time.

Reply:
No worries, our front-end always will get the dates.